src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Models/EmployerRequests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using SFA.DAS.ProviderRequestApprenticeTraining.Application.Queries.GetEmployerRequestsByIds;
+using SFA.DAS.ProviderRequestApprenticeTraining.Application.Queries.GetSelectEmployerRequests;
+using SFA.DAS.ProviderRequestApprenticeTraining.InnerApi.Responses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Api.Models
+{
+    public class EmployerRequests
+    {
+        public string StandardReference { get; set; }
+        public string StandardTitle { get; set; }
+        public int StandardLevel { get; set; }
+        public List<EmployerRequest> Requests { get; set; }
+
+        public static implicit operator EmployerRequests(GetEmployerRequestsByIdsResult source)
+        {
+            if (source.EmployerRequests.Any())
+            {
+                return new EmployerRequests
+                {
+                    StandardReference = source.EmployerRequests.FirstOrDefault().StandardReference,
```

Comment:
This was a Sonar cloud warning in another place - it should be First() as you have already done Any()

Reply:
Also updated in SelectEmployerRequests as I had done the same there